### DISCUSSION TITLE
Welcome to resumake.io Discussions!

### DISCUSSION THREAD

**Discussion by saadq** [saadq/resumake.io#163] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fullstacc** [comment]
Hi @saadq , I forked your v3 branch but can't seem to get the "refresh" feature to work after filling out a form.
Is this functionality still in development?

For added context, I ran "npm setup" followed by "npm start"

**Comment by saadq** [comment]
Hey @sulfuroussquares sorry for the late response. So I am actually in the process of doing a re-write of my re-write 😅

Most of the `v3` progress is actually in the `old-v3` branch. That's where you should be able to see stuff like custom sections and auto-regeneration. If you do want to try running that, you can either use `docker-compose up -d` if you have docker installed or you can just do `npm start`, but if you choose to use `npm` just make sure to update the `"proxy"` key inside the `client/package.json` file and set it to `localhost:4001` instead of `server:4001`.

**Comment by fullstacc** [comment]
Hi @saadq ! Sorry for also being a little bit behind; I'd like to start attempting some issue fixes but I must follow back up and ask:
- Is `old-v3` still the most up-to-date branch? The commit history implies that I should branch from `main`
- If I were to branch from `main`, I noticed that after doing a basic `npm i` and then `npm run dev` like the readme states, I'm getting `MODULE_NOT_FOUND` errors coming from `resumake.io/.next/server/pages/api/generate/pdf.js:97:3)`. Is there another setup step I'm missing?

**Reply by ducaale** [reply]
Hi @fullstacc, can you check which node version you are using? the current requirement is v16 or greater. Can you also post the whole stack trace? thanks
